src/main/java/com/epam/flightscontroltool/boundary/dto/AirportDto.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.flightscontroltool.boundary.dto;
+
+import com.epam.flightscontroltool.control.mapper.AirportMapper;
+import com.epam.flightscontroltool.entity.Airport;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class AirportDto {
+
+    @JsonProperty("id")
```

Comment:
Usually, we use this annotation to override DTO property name. If DTO property name and target name are the same no need to specify it. It's gonna work without that annotation, better to remove it.